src/collective/workspace/membership.py

```diff
@@ -1,18 +1,19 @@
+# coding=utf-8
 from BTrees.Length import Length
 from collective.workspace.events import TeamMemberModifiedEvent
 from collective.workspace.events import TeamMemberRemovedEvent
 from collective.workspace.interfaces import _
 from collective.workspace.interfaces import IWorkspace
 from collective.workspace.pas import add_group
 from collective.workspace.pas import get_workspace_groups_plugin
-from collective.workspace.vocabs import UsersSource
 from copy import deepcopy
 from DateTime import DateTime
 from plone import api
+from plone.app.z3cform.widget import AjaxSelectFieldWidget
 from plone.autoform import directives as form
-from plone.formwidget.autocomplete import AutocompleteFieldWidget
 from plone.supermodel import model
 from plone.uuid.interfaces import IUUIDGenerator
+from Products.CMFPlone.utils import safe_nativestring
```

Comment:
@cillianderoiste @ale-rt this line breaks compatibility with all Plone versions except 5.2. Do we really want to make c.workspace a Plone 5.2 only release? https://travis-ci.org/collective/collective.workspace/builds/641340682?utm_medium=notification&utm_source=github_status

Reply:
That is what is written in the `setup.py` support only 5.2.
I do not see any bad thing in that.
We could support also 5.1 with something like
https://github.com/plone/Products.PortalTransforms/blob/b739daa3a34d2803e82db2baf2f2c8e9ca8374b7/Products/PortalTransforms/utils.py#L8-L22
But it will be harder to support previous versions because of the switch to named behaviors.

Reply:
Actually Plone 5.1 compatibility is more than that, see https://travis-ci.org/collective/collective.workspace/jobs/641351164?utm_medium=notification&utm_source=github_status